server/service/individualServices/RegardApplication.js

```diff
@@ -99,13 +99,13 @@ exports.regardApplication = function (applicationName, releaseNumber, user, xCor
                             } else if(attempts == maximumNumberOfAttemptsToCreateLink && result['status'] != 200){
                                 resolve({
                                     "successfully-connected": false,
-                                    "reason-of-failure": "OL_UNKNOWN"
+                                    "reason-of-failure": "OL_DID_NOT_REACH_ALT"
```

Comment:
if (result['status'] != 200) , check whether the status is 408 , then mention that OL did not reach ALT